pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -1172,7 +1175,6 @@ type OVHCloudSDConfig struct {
 	// +required
 	ConsumerKey corev1.SecretKeySelector `json:"consumerKey"`
 	// Service of the targets to retrieve. Must be `VPS` or `DedicatedServer`.
-	// +kubebuilder:validation:Enum=VPS;DedicatedServer
```

Comment:
any reason removing this?
was linter pointing out issues?

Reply:
Yes. 
```
$ make check-golang  
......
INFO [runner] linters took 103.027459ms with stages: goanalysis_metalinter: 102.243291ms 
pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go:1180:2: duplicatemarkers: Service has duplicated markers kubebuilder:validation:Enum=VPS;DedicatedServer (kubeapilinter)
	Service OVHService `json:"service"`
	^
1 issues:
* kubeapilinter: 1
INFO File cache stats: 1 entries of total size 62.0KiB 
INFO Memory: 10 samples, avg is 46.5MB, max is 116.5MB 
INFO Execution took 857.9ms                       
make: *** [check-golang] Error 1
```